keyboards/matchstickworks/southpad/rules.mk

```diff
@@ -0,0 +1,25 @@
+# MCU name
+MCU = atmega32u2
+
+# Processor frequency
+F_CPU = 16000000
+
```

Comment:
```suggestion
```
Whoops, this actually isn't needed at all; the default frequency for the AVRs (with one exception) is 16MHz.

Reply:
Ah! Wasn't sure if QMK knew to make that assumption. Good to know for the future!